packages/ui-react-utils/src/omitProps.ts

```diff
@@ -67,7 +67,7 @@ const omit = <T>(originalObject: T, keysToOmit: string[]) => {
       key === 'styles' ||
       key === 'makeStyles' ||
       key === 'themeOverride' ||
-      key === 'instanceCounterMap'
```

Comment:
Shouldn't it be both? We shouldn't accidentally pass `instanceCounterMap` either, or?

Reply:
No, since `instanceCounterMap` is not passed down to any component anymore, just the `deterministicId` function.
https://github.com/instructure/instructure-ui/blob/master/packages/ui-react-utils/src/DeterministicIdContext/withDeterministicId.tsx#L39-L76

Reply:
oh cool, thanks :)